views/components/card.html

```diff
@@ -0,0 +1,12 @@
+{% macro card(data) %}
+<article class="card" itemscope itemtype="http://schema.org/Product">
```

Comment:
The items in here don't always semantically contain a product. In fact, the sample data you have here specifically isn't a product. This doesn't necessarily need any more specific type of schema other than `thing` if we can't guarantee the content.